monitoring/storage.go

```diff
@@ -50,9 +88,15 @@ type HighWatermarkCheckerData struct {
 	AvailableBytes uint64 `json:"available_bytes"`
 }
 
-// FailureMessage returns failure watermark check message
-func (d HighWatermarkCheckerData) FailureMessage() string {
-	return fmt.Sprintf("disk utilization on %s exceeds %v percent (%s is available out of %s), see https://gravitational.com/telekube/docs/cluster/#garbage-collection",
+// WarningMessage returns warning watermark check message
+func (d HighWatermarkCheckerData) WarningMessage() string {
+	return fmt.Sprintf("disk utilization on %s exceeds %v percent, cluster will degrade if usage exceeds %v percent (%s is available out of %s), see https://gravitational.com/gravity/docs/cluster/#garbage-collection",
```

Comment:
Nit: I'd move `(%s is available out of %s)` after the first part of the sentence, otherwise it reads a bit confusing:

```
disk utilization on %s exceeds %v percent (%s is available out of %s), cluster will degrade if usage exceeds %v percent, see https://gravitational.com/gravity/docs/cluster/#garbage-collection
```